bip-0341.mediawiki

```diff
@@ -136,6 +136,9 @@ In summary, the semantics of the [[bip-0143.mediawiki|BIP143]] sighash types rem
 
 ==== Taproot key path spending signature validation ====
 
+A Taproot signature is the 64-byte Schnorr signature defined in [[bip-0340.mediawiki|BIP340]] with the sighash byte appended in the usual Bitcoin fashion<ref>See [https://en.bitcoin.it/wiki/OP_CHECKSIG#How_it_works '''OP_CHECKSIG''' "How it works"].</ref>.
```

Comment:
I'm not very keen on linking to the bitcoin.it wiki, especially not without a fixed timestamp. It's hard to say the OP_CHECKSIG article there will forever include an explanation of this concept.

Reply:
I will look elsewhere in the BIPs for a proper reference.

Reply:
I don't think you'll find any, since the "append a byte to the ECDSA signature to indicate the sighash type" dates back to the original release, long before the BIP process. I'd just leave the text without a reference.

Reply:
ok.